src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -65,7 +65,8 @@ public PersistentDataStore() {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, creationTime);
+ 	//To DO: get Password from Datastore; Currently null
```

Comment:
Comment isnt aligned with the rest of the code. Leave a space afte the //. And fix the capitalization of the TODO, keep it consistent with what you did above or change both to TODO.